pnpm-lock.yaml

```diff
@@ -3194,20 +3194,20 @@ packages:
     engines: {node: '>= 12.0.0'}
     dependencies:
       '@aws-sdk/types': 3.55.0
-      tslib: 2.3.1
+      tslib: 2.4.0
```

Comment:
There is not a package change in this PR so not sure why is there a version change in the lock file.